test/kernels/keops/test_matern_kernel.py

```diff
@@ -19,34 +21,56 @@ def create_kernel_ard(self, num_dims, **kwargs):
             return MaternKernel(nu=2.5, ard_num_dims=num_dims, **kwargs)
 
     class TestMaternKeOpsKernel(unittest.TestCase):
```

Comment:
Unfortunately these tests won't run on our CI (which only runs on CPU)... they will only be run locally.

Reply:
I have removed the `.cuda()` from these tests, so they should run in the CI now.